pkg/virt-handler/vm.go

```diff
@@ -2658,7 +2658,11 @@ func (d *VirtualMachineController) vmUpdateHelperMigrationTarget(origVMI *v1.Vir
 	}
 
 	// configure network inside virt-launcher compute container
-	if err := d.setupNetwork(vmi, vmi.Spec.Networks); err != nil {
+	nonAbsentIfaces := netvmispec.FilterInterfacesSpec(vmi.Spec.Domain.Devices.Interfaces, func(iface v1.Interface) bool {
+		return iface.State != v1.InterfaceStateAbsent
+	})
+	nonAbsentNets := netvmispec.FilterNetworksByInterfaces(vmi.Spec.Networks, nonAbsentIfaces)
```

Comment:
Per the findings of @ormergi when testing old VMI/s with an interface marked for removal and a migration, this condition causes the migration to fail.
The target will not process an interface marked for removal, even if the VMI is "old".

The required information about the VMI being old or new is not available at this stage, therefore, we have two options:
- Drop this filter here and allow it to be created anyway. Optimize for it in a following change that cleans up the pod networking configuration.
- Move this filtering from here and move it deeper into the setup where the information is available and skip it there.

Reply:
Fixed using the 2nd option mentioned above.